apps/team-awesome-web/src/components/__tests__/.Select.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+
+import { Select } from "../forms/select";
+
+it("If the select updates its options", async () => {
+  const sortList = "timesheets";
+
+  const utils = render(<Select sortList={sortList} />);
+
+  expect((utils.getByRole("option", { name: "client" }) as HTMLOptionElement).selected).toBe(true);
+
+  fireEvent.change(utils.getByRole("combobox"), {
+    target: { name: sortList[1] },
+  });
+
+  expect((utils.getByRole("option", { name: "date" }) as HTMLOptionElement).selected).toBe(true);
+});
```

Comment:
Nice man, Good you made this work, i made a simpler test.

Reply:
It didn't though, for fast switching I built in that filenames preceded with a dot would be ignored 😅  Will remove this test to avoid confusion.